apps/src/dance/ai/utils/index.ts

```diff
@@ -1,9 +1,13 @@
+export * from './chooseEffects';
```

Comment:
Curious how @fisher-alice and @tshaffercodeorg feel about this - in theory I like modularizing util functions where possible, but the set of AI generating functions really only used by each other right? Do we think we might use some of these other functions in different contexts? If not, then I think it would helpful to enforce that `chooseEffects` is the only public entrypoint into these AI functions and keep them in one file.

Reply:
I think that keeping `chooseEffects` and `getGeneratedEffectScores` as the only functions that can be exported makes sense. @thomasoniii  [originally discussed](https://github.com/code-dot-org/code-dot-org/pull/55089#discussion_r1401997592) these util function files as buckets of functions so keeping these in a single bucket seems like a good call and would make sense to a future reader.

Reply:
Cool, yeah I'd also agree that keeping only those two functions exported would make sense